src/main/java/com/google/sps/data/DataUtil.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+import com.google.gson.Gson;
+
+public class DataUtil {
+    public static final Gson PARSER = new Gson();
+    public static enum video_state {
+        UNSTARTED(-1), ENDED(0), PLAYING(1), PAUSED(2), BUFFERING(3), CUED(5);
```

Comment:
nit: add comments to each of them so your pod mates and other contributors in general know when to use them.